src/shared/components/lazyMobXTable/LazyMobXTable.tsx

```diff
@@ -595,6 +617,9 @@ export default class LazyMobXTable<T> extends React.Component<LazyMobXTableProps
                 <Observer>
                     {this.getTable}
                 </Observer>
+                <Observer>
+                    {this.showMore}
```

Comment:
Seems like this is wrapped in two `Observer` tags, I don't think you really need the `showMore` function.